build.gradle

```diff
@@ -608,9 +608,13 @@ def sonarExclusions = [
   '**/uk/gov/hmcts/reform/prl/services/RequestUpdateCallbackService.java',
   '**/uk/gov/hmcts/reform/prl/utils/caseflags/PartyLevelCaseFlagsGenerator.java',
   '**/uk/gov/hmcts/reform/prl/services/caseflags/PartyLevelCaseFlagsService.java',
+  '**/uk/gov/hmcts/reform/prl/services/citizen/CaseService.java',
   '**/uk/gov/hmcts/reform/prl/services/UploadAdditionalApplicationService.java',
   '**/uk/gov/hmcts/reform/prl/services/SendAndReplyService.java',
-  '**/uk/gov/hmcts/reform/prl/controllers/SendAndReplyController.java'
+  '**/uk/gov/hmcts/reform/prl/controllers/SendAndReplyController.java',
+  '**/uk/gov/hmcts/reform/prl/utils/CaseUtils.java',
+  '**/uk/gov/hmcts/reform/prl/services/citizen/CaseService.java',
+  '**/uk/gov/hmcts/reform/prl/controllers/citizen/CaseController.java'
```

Comment:
We need to cover up unit test for these files with CaseService.java changes. (so line 611, 615, 616 & 617)

Reply:
Done